src/backend/types.js

```diff
@@ -32,6 +32,16 @@ export type HookType =
   | 'useImperativeHandle'
   | 'useDebugValue';
 
+// Priority level is copied from React and should be kept in sync:
+// https://github.com/facebook/react/blob/master/packages/react-reconciler/src/SchedulerWithReactIntegration.js
+export opaque type ReactPriorityLevel = 99 | 98 | 97 | 96 | 95 | 90;
+export const ImmediatePriority: ReactPriorityLevel = 99;
+export const UserBlockingPriority: ReactPriorityLevel = 98;
+export const NormalPriority: ReactPriorityLevel = 97;
+export const LowPriority: ReactPriorityLevel = 96;
+export const IdlePriority: ReactPriorityLevel = 95;
+export const NoPriority: ReactPriorityLevel = 90;
```

Comment:
Note to self: These would need to live in the renderer interface and be assigned based on renderer version. This was just a quick hack. :)